Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteNonQueryViewModel.cs

```diff
@@ -86,37 +73,36 @@ protected override void InitializeModel()
             int propertyOrderIndex = 1;
 
             ExistingDbConnection.IsPrincipal = true;
+            ExistingDbConnection.IsRequired = true;
             ExistingDbConnection.OrderIndex = propertyOrderIndex++;
             ExistingDbConnection.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
 
-            Sql.IsPrincipal = true;
-            Sql.IsRequired = true;
-            Sql.OrderIndex = propertyOrderIndex++;
-            Sql.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
-
-            ConnectionString.OrderIndex = propertyOrderIndex++;
-            ConnectionString.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
-
-            ProviderName.OrderIndex = propertyOrderIndex++;
-            ProviderName.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
-
-            ConnectionSecureString.OrderIndex = propertyOrderIndex++;
-            ConnectionSecureString.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
-
             CommandType.OrderIndex = propertyOrderIndex++;
+            CommandType.IsPrincipal = true;
+            CommandType.IsRequired = true;
+            CommandType.DataSource = DataSourceBuilder<CommandType>
+                .WithId(t => t.ToString())
+                .WithLabel(t => t.GetLabel())
+                .WithData(Enum.GetValues(typeof(CommandType)) as IReadOnlyList<CommandType>)
+                .Build();
             CommandType.Widget = new DefaultWidget { Type = ViewModelWidgetType.Dropdown };
 
-            TimeoutMS.OrderIndex = propertyOrderIndex++;
-            TimeoutMS.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
+            Sql.IsPrincipal = true;
+            Sql.IsRequired = true;
+            Sql.OrderIndex = propertyOrderIndex++;
+            Sql.Widget = new DefaultWidget { Type = ViewModelWidgetType.TextComposer };
 
             Parameters.OrderIndex = propertyOrderIndex++;
             Parameters.Widget = new DefaultWidget { Type = ViewModelWidgetType.Dictionary };
 
-            AffectedRecords.OrderIndex = propertyOrderIndex++;
-            AffectedRecords.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
+            TimeoutMS.OrderIndex = propertyOrderIndex;
```

Comment:
```suggestion
            TimeoutMS.OrderIndex = propertyOrderIndex++;
```

Reply:
In the spec, `timeout` and `continue on error` properties are on the same line